.github/workflows/run-mocha-tests.yml

```
@@ -11,11 +11,16 @@ on:
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
-        node-version: [20.x, 22.x]
+        os: [ubuntu-latest, windows-2022]
+        node-version: [20.x, 22.x, 24.x]
+
+    strategy:
```

actionlint (1.7.12)

[error] key "strategy" is duplicated in "build" job. previously defined at line:16,col:5

(syntax-check)